### PR TITLE
Create metric card for METEOR

### PR DESCRIPTION
**Pull request by sashavor**
Proposing a metric card for METEOR

### CODE REVIEW

File: metrics/meteor/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+```python
+>>> from datasets import load_metric
+>>> meteor = load_metric('meteor')
+>>> predictions = ["It is a guide to action which ensures that the military always obeys the commands of the party"]
+>>> references = ["It is a guide to action that ensures that the military will forever heed Party commands"]
+>>> results = meteor.compute(predictions=predictions, references=references)
+```
+
+You will need [NLTK](https://www.nltk.org/) version >=3.7 for METEOR score to work.
```
Comment (albertvillanova) [inline metrics/meteor/README.md:36]: Are you sure about this requirement? Looking at the script code, we did some fixes to make it work for different NLTK versions, like 3.6.4, 3.6.5, 3.6.6...

File: metrics/meteor/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Maximal values :
+
+```python
+>>> from datasets import load_metric
+>>> meteor = load_metric('meteor')
+>>> predictions = ["It is a guide to action which ensures that the military always obeys the commands of the party"]
+>>> references = ["It is a guide to action which ensures that the military always obeys the commands of the party"]
+>>> results = meteor.compute(predictions=predictions, references=references)
+>>> print(round(results['meteor'],2))
```
Comment (albertvillanova) [inline metrics/meteor/README.md:57]: Style nit.
```suggestion
>>> print(round(results['meteor'], 2))
```

File: metrics/meteor/README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+Minimal values:
+
+```python
+>>> from datasets import load_metric
+>>> meteor = load_metric('meteor')
+>>> predictions = ["It is a guide to action which ensures that the military always obeys the commands of the party"]
+>>> references = ["Hello world"]
+>>> results = meteor.compute(predictions=predictions, references=references)
+>>> print(round(results['meteor'],2))
```
Comment (albertvillanova) [inline metrics/meteor/README.md:69]:
```suggestion
>>> print(round(results['meteor'], 2))
```

File: metrics/meteor/README.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+Partial match:
+
+```python
+>>> from datasets import load_metric
+>>> meteor = load_metric('meteor')
+>>> predictions = ["It is a guide to action which ensures that the military always obeys the commands of the party"]
+>>> references = ["It is a guide to action that ensures that the military will forever heed Party commands"]
+>>> results = meteor.compute(predictions=predictions, references=references)
+>>> print(round(results['meteor'],2))
```
Comment (albertvillanova) [inline metrics/meteor/README.md:81]:
```suggestion
>>> print(round(results['meteor'], 2))
```

**Review by albertvillanova — APPROVED** [review]
Thanks!!! :hugs:

File: metrics/meteor/README.md (hunk 1 above)
Comment (sashavor) [reply metrics/meteor/README.md]: it wasn't working for me when I had version 3.6.4...

**Pull request merged**